### PR TITLE
Feature/anti spam protection

### PR DESCRIPTION
**Pull request by Xarlos89**
Anti spam, what u think?

### CODE REVIEW

File: src/zorak/cogs/admin/admin_automod_spam_messages.py [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+from zorak.utilities.cog_helpers._embeds import (
+    embed_spammer,  # pylint: disable=E0401
+    embed_spammer_warn  # pylint: disable=E0401
+)
+
+
+logger = logging.getLogger(__name__)
+
+
+def is_second_message(first, second):
```
Comment (JamMassey) [inline src/zorak/cogs/admin/admin_automod_spam_messages.py]: pedantic; unneeded abstraction

**Comment by itzhapp** [comment]
Maybe a small optimization could be hashing message.content cause not using the message in anyway other then comparing

File: src/zorak/cogs/admin/admin_automod_spam_messages.py (hunk 1 above)
Comment (Xarlos89) [reply src/zorak/cogs/admin/admin_automod_spam_messages.py]: True, there were 2 iterations before this and this might be some leftover.

Comment (Xarlos89) [reply src/zorak/cogs/admin/admin_automod_spam_messages.py]: Lol, wasn't even used in the code. Was just old floating trash

**Comment by Xarlos89** [comment]
> Maybe a small optimization could be hashing message.content cause not using the message in anyway other then comparing

Created a new issue for this. Might be good for zorakathon

**Pull request merged**